grails-app/assets/javascripts/directives/streama-video-player-directive.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 streamaApp.directive('streamaVideoPlayer', [
-  'uploadService', 'localStorageService', '$timeout',
-  function (uploadService, localStorageService, $timeout) {
+  'uploadService', 'localStorageService', '$timeout','playerService',
```

Comment:
There should be a space behind the ,  . This is also something that the editorconfig will take of for you.